client/components/pages/files/EditNameDialog.vue

```diff
@@ -0,0 +1,126 @@
+<script lang="ts" setup>
+import { object, string } from 'yup'
+import { useEditFileName } from '~/composables/fileTable/useChangeFileName'
+import { UPDATE_FILE_FORM } from '~/utils/forms'
```

Comment:
same (?)

Reply:
No, this one is unfortunately mandatory.

Reply:
I'll do something about it later